### ISSUE TITLE
Building fire detection models using tensor flow in python (Machine learning)

### ISSUE THREAD

**Issue by vitalisodhis** [section-engineering-education/engineering-education#7727]
### Topic Suggestion
Building fire detection models using tensor flow in python (Machine learning)

### Writing sample(s): 
- https://www.section.io/engineering-education/image-processing-using-matlab/
- https://www.section.io/engineering-education/organizing-execution-of-programs-using-timer-object-in-matlab/

#### Proposed article introduction
Fire can lead to lost lives and the destruction of valuable properties when out of control.
Therefore, several measures, such as smoke detectors, are used to detect fire presence.
This article will discuss how to train a model using python to differentiate between images with fire and images without fire. Python has packages that are installed in the working environment. Different  Packages perform different functions which help in the tasks. Training of the fire detection model will be done using packages: TensorFlow, Pandas, Matlib, Seaborn, Plotly, and Keras.
 
The training will be done using a dataset with two folders: Fire images (containing 755 images) and non-fire images (containing 244 images). Extensive data are used while training models to increase the model's accuracy.

#### Key takeaways 
- Model training in python.
-  Handling large image data.
- Image classification in machine learning.
- Model testing.

#### Article quality
In this article, the model is trained to detect fire by comparing  the current situation with a previous fire situation. Many fire and non-fire images are  used for training the model to differentiate between fire an non fire situation, the output of the detection is then assigned a binary value (1 or 0).

**Comment by hectorkambow** [comment]
@vitalisodhis 
Sounds like a helpful topic - lets please be sure it adds value beyond what is in any official docs and/or what is covered in other blog sites. (We're really looking for unique/original content. EngEd articles should go beyond a basic explanation - and it is always best to reference any EngEd article and build upon it).

Please be attentive to grammar/readability and make sure that you put your article through a thorough editing review prior to submitting for final approval. (There are some great free tools that we reference in EngEd resources.)

ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.

Please be sure to double check that it does not overlap with any existing EngEd articles, articles on other blog sites, or any incoming EngEd topic suggestions (if you haven't already) to avoid any potential article closure.
Please reference any relevant EngEd articles in yours. - Approved
